### PR TITLE
CORTX-32914: Update ova-solution.yaml

### PR DESCRIPTION
**Pull request by mukul-seagate11**
Signed-off-by: Mukul Malhotra <mukul.malhotra@seagate.com>

# Problem Statement
- Update for ova solution as per v0.9.0 with updated resource limits as validated

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing
https://eos-jenkins.colo.seagate.com/job/Cortx-kubernetes/job/setup-cortx-cluster-solution-input/681/console

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: solutions/ova/kubernetes/ova-solution.yaml [hunk 1]
```diff
@@ -87,102 +76,103 @@ solution:
...
-    node1:
-      name: cortx-ova-rgw.seagate.com
+              memory: 512Mi
+              cpu: 250m
+  storage_sets:
+    - name: storage-set-1
+      durability:
+        sns: 1+0+0
+        dix: 1+0+0
+      container_group_size: "1"
```
Comment (gauravchaudhari02) [inline solutions/ova/kubernetes/ova-solution.yaml:164]: do we need double quotes around number here?

File: solutions/ova/kubernetes/ova-solution.yaml [hunk 2]
```diff
@@ -87,102 +76,103 @@ solution:
...
+            metadata:
+              - path: /dev/sdb
+                size: 5Gi
+            data:
+              - path: /dev/sdc
+                size: 5Gi
+              - path: /dev/sdd
+                size: 5Gi
+      nodes:
+        - cortx-ova-rgw.seagate.com
```
Comment (gauravchaudhari02) [inline solutions/ova/kubernetes/ova-solution.yaml:178]: can we rename hostname to `cortx-rgw-ova.seagate.com`

Comment (shailesh-vaidya) [reply solutions/ova/kubernetes/ova-solution.yaml:178]: Actually we should remove `rgw` now

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: solutions/ova/kubernetes/ova-solution.yaml [hunk 3]
```diff
@@ -65,19 +54,19 @@ solution:
           storage: 10Gi
           resources:
             requests:
-              memory: 200Mi
-              cpu: 200m
+              memory: 1Gi
```
Comment (madhavemuri) [inline solutions/ova/kubernetes/ova-solution.yaml:57]: I think for OVA we can keep the old values for consul server.

File: solutions/ova/kubernetes/ova-solution.yaml [hunk 4]
```diff
@@ -87,102 +76,103 @@ solution:
...
+            memory: 2Gi
+            cpu: 500m
       hare:
         hax:
           resources:
             requests:
-              memory: 128Mi
-              cpu: 250m
+              memory: 256Mi
+              cpu: 400m
```
Comment (madhavemuri) [inline solutions/ova/kubernetes/ova-solution.yaml:95]: cpu for hax too can be 250m.

File: solutions/ova/kubernetes/ova-solution.yaml [hunk 5]
```diff
@@ -87,102 +76,103 @@ solution:
...
             limits:
-              memory: 2Gi
-              cpu: 1000m
+              memory: 768Mi
+              cpu: 800m
```
Comment (madhavemuri) [inline solutions/ova/kubernetes/ova-solution.yaml:98]: Also for higher limits use 400m for hax.

**Comment by mukul-seagate11** [comment]
@all, suggested comments will be addressed in new PR

**Pull request closed without merging**